x/oracle/types/msgs_test.go

```diff
@@ -94,9 +94,9 @@ func TestMsgAggregateExchangeRateVote(t *testing.T) {
 	msgInvalidUnknownRequest := "must provide at least one oracle exchange rate: unknown request"
 	msgInvalidFeederAddr := "invalid feeder address (empty address string is not allowed): invalid address"
 	msgInvalidOperatorAddr := "invalid operator address (empty address string is not allowed): invalid address"
-	msgInvalidOraclePrice := "failed to parse exchange rates string cause: invalid oracle price: invalid coins"
+	msgInvalidOraclePrice := "oracle price"
```

Comment:
These error messages seem a lot less descriptive. I don't see where these changed in the code, was it always this way?

Reply:
The error messages in the module are actually more descriptive now; there are just multiple different cases
But, they all include this text